src/ray/common/task/task_spec.cc

```diff
@@ -372,17 +378,21 @@ std::string TaskSpecification::CallSiteString() const {
 
 WorkerCacheKey::WorkerCacheKey(
     const std::unordered_map<std::string, std::string> override_environment_variables,
-    const std::string serialized_runtime_env)
+    const std::string serialized_runtime_env,
+    const std::unordered_map<std::string, double> required_resources)
```

Comment:
I think `map` is better than `unorderedmap` here, because we need iterate the items to compute the hashvalue.

Reply:
There is no need `const` on pass-by-copy arguments.